### PR TITLE
Tests for server/httpserver/config

### PR DESCRIPTION
**Pull request by LopatkinEvgeniy**

### CODE REVIEW

**Comment by FZambia** [comment]
Sorry for delay - actually I commented a month ago but have not noticed that I have to push submit review button - and all my comments were invisible for all that time:)

File: libcentrifugo/server/httpserver/config_test.go [hunk 1]
```diff
@@ -1 +1,125 @@
...
+	assert.True(t, stringInSlice("one", []string{"one", "two", "tree"}))
+	assert.True(t, stringInSlice("two", []string{"one", "two", "tree"}))
+	assert.True(t, stringInSlice("", []string{"one", "two", "tree", ""}))
+
+	assert.False(t, stringInSlice("four", []string{"one", "two", "tree"}))
+	assert.False(t, stringInSlice("ab", []string{"aba", "bab"}))
+	assert.False(t, stringInSlice("four", []string{}))
+	assert.False(t, stringInSlice("", []string{}))
+	assert.False(t, stringInSlice("", []string{"one", "two", "tree"}))
+}
```
Comment (LopatkinEvgeniy) [inline libcentrifugo/server/httpserver/config_test.go]: I deleted this

File: libcentrifugo/server/httpserver/config_test.go [hunk 2]
```diff
@@ -1 +1,125 @@
...
+	c.SSLKey = fakeKey
+	assert.Error(t, c.Validate())
+
+	c.SSLCert = fakeSert
+	c.SSLKey = ""
+	assert.Error(t, c.Validate())
+}
+
+const (
+	SOCKSJS_HEARDBEAT_DELAY = iota
```
Comment (LopatkinEvgeniy) [inline libcentrifugo/server/httpserver/config_test.go]: fixed

File: libcentrifugo/server/httpserver/config_test.go [hunk 3]
```diff
@@ -1 +1,125 @@
...
+	assert.Equal(t, cfg.SSLAutocertServerName, "ssl_autocert_server_name_value")
+	assert.True(t, cfg.WebsocketCompression)
+	assert.Equal(t, cfg.WebsocketCompressionLevel, WEBSOCKET_COMPRESSION_LEVEL)
+	assert.Equal(t, cfg.WebsocketCompressionMinSize, WEBSOCKET_COMPRESSION_MIN_SIZE)
+	assert.Equal(t, cfg.WebsocketReadBufferSize, WEBSOCKET_READ_BUFFER_SIZE)
+	assert.Equal(t, cfg.WebsocketWriteBufferSize, WEBSOCKET_WRITE_BUFFER_SIZE)
+
+	cfg = newConfig(&fakeConfigGetter{autocertHostWhitelist: "one,two,three"})
+	assert.Equal(t, cfg.SSLAutocertHostWhitelist, []string{"one", "two", "three"})
+}
```
Comment (LopatkinEvgeniy) [inline libcentrifugo/server/httpserver/config_test.go]: ok

**Pull request merged**